tests/local/component-model/resources.wast

```diff
@@ -1112,3 +1112,18 @@
     ))
   )
   "resource types are not the same")
+
+(component
```

Comment:
Just a thought - we have git history to tell when these tests were added, but it would probably be helpful to document what these are specifically testing for.